lib/tasks/db_tasks.rake

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'active_record'
+require 'active_support'
+
+namespace :db do
+  namespace :migrate do
+    desc "Migrates the database by adding partition_guid and encryption_epoch to the specified model (must specify :model)"
+    task :add_encryption_fields, [:model] => :environment do |_, args|
+      model = args[:model].camelize
+
+      abort 'Need to provide `model` as argument' if model.blank?
+
+      add_column model, :partition_guid, :string
+      add_column model, :encryption_epoch, :datetime
+
+      model_camelcase = model.to_s.camelize.singularize.constantize
+
+      model_camelcase.reset_column_information
+      model_camelcase.find_each do |record|
+        record.generate_partition_guid
+        record.generate_encryption_epoch
+
+        record.save!(validate: false)
+      end
+
+      change_column model, :partition_guid, :string, null: false
+      change_column model, :encryption_epoch, :datetime, null: false
```

Comment:
This is the only nontrivial change so far from the latest analogous PR for porkylib that Ben had up.  This is in response to a comment on the wile PR (https://github.com/Zetatango/wile_e/pull/992/) that we have to first populate existing records before setting the new column to mandatory.

Reply:
Ok, so turns out this doesn't actually work directly typed into a rakefile.  It would have to generate a migration that has this code as text, and from talking to @dragoszt this may be more effort than it's worth (writing a custom generator).  Perhaps instead we just give a template in the README file for this?